src/pybind/mgr/dashboard/frontend/src/app/shared/services/api-interceptor.service.ts

```diff
@@ -36,6 +37,10 @@ export class ApiInterceptorService implements HttpInterceptor {
   intercept(request: HttpRequest<any>, next: HttpHandler): Observable<HttpEvent<any>> {
     return next.handle(request).pipe(
       catchError((resp: CdHttpErrorResponse) => {
+        if (resp.headers.has(interceptorSkipHeader)) {
+          resp.preventDefault();
+          return observableThrowError(resp);
+        }
```

Comment:
You are adding the header to the request and not in the response.
`preventDefault` is only defined below this line, so you can't call it and its not needed here.
I removed the `observableThrowError` so it doesn't throw an error on the browser console and since this is handle by us, I don't think it should be considered an error.
```suggestion
        if (request.headers.has(interceptorSkipHeader)) {
          return of(null);
        }
```

Reply:
That makes sense. Thank you :)